### PR TITLE
Use the Kubeflow DockerHub organization for the GitHub action for issue_triage

### PR DESCRIPTION
**Pull request by jlewi**
* Instead of storing the image in a personal account we should use the kubeflow
  organization.

* Update the issue_triage.yaml workflow in this repository to use the master version of the action

* Related to kubeflow/code-intelligence#66
* Related to kubeflow/community#27 - GitHub action for adding issues to triage

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/code-intelligence/67)

### CODE REVIEW

**Comment by hamelsmu** [comment]
@jlewi you also need to change .github/workflows/triage_issues.yaml

While you are at it can you add a more verbose explanation to `Issue_Triage/action/README.md` so that the community can understand what business rules either add or remove an issue from the Kanban board?

File: .github/workflows/build_image.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ jobs:
         run: |
           cd $GITHUB_WORKSPACE
           echo ${PASSWORD} | docker login -u $USERNAME --password-stdin
-          docker build -t hamelsmu/kubeflow-triage -f Issue_Triage/Dockerfile .
-          docker push hamelsmu/kubeflow-triage
+          docker build -t kubeflow/kubeflow-triage -f Issue_Triage/Dockerfile .
+          docker push kubeflow/kubeflow-triage
         env:
           USERNAME: ${{ secrets.DOCKER_USERNAME }}
           PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```
Comment (hamelsmu) [inline .github/workflows/build_image.yaml]: @jlewi confirming that you changed the secrets in this repo for both `DOCKER_USERNAME` and `DOCKER_PASSWORD` OR you added me to the Kubeflow Org  (Either one would work)

Comment (jlewi) [reply .github/workflows/build_image.yaml]: Changed DOCKER_USERNAME and DOCKER_PASSWORD.

File: Issue_Triage/action/README.md [hunk 2]
```diff
@@ -35,4 +35,18 @@ jobs:
...
 
+### Installing the action on a repository
 
+1. Create a workflow like the one above in your repository in the directory
+
+   ```
+   .github/workflows/
+   ```
+
+1. Create a secret for the repository called `triage_projects_github_token` which has a GitHub personal access token with the following permissions
```
Comment (hamelsmu) [inline Issue_Triage/action/README.md]:
```suggestion
1. [Create a secret for the repository](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) called `triage_projects_github_token` which has a GitHub personal access token with the following permissions
```

File: Issue_Triage/action/README.md [hunk 3]
```diff
@@ -35,4 +35,18 @@ jobs:
 2. **ISSUE_NUMBER**: The issue number in the current repo that you want to triage
 3. **GITHUB_PERSONAL_ACCESS_TOKEN**: A personal access token with authorization to modify the project board.
 
+### Installing the action on a repository
 
+1. Create a workflow like the one above in your repository in the directory
+
+   ```
+   .github/workflows/
```
Comment (hamelsmu) [inline Issue_Triage/action/README.md]:
```suggestion
   .github/workflows/<your_file_name>.yaml
```

**Comment by hamelsmu** [comment]
@jlewi the push to DockerHub seems to be failing `denied: requested access to the resource is denied` https://github.com/kubeflow/code-intelligence/commit/5cc4c0a24528440824653b8ccc9fe6db7394fdc4/checks?check_suite_id=328647868#step:3:154

**Comment by hamelsmu** [comment]
Ohhhhh ! !!!! I know why b/c you are opening this PR from a Fork.   Yeah you cannot trigger a build from a Fork because your Fork doesn't have access to secrets and you cannot get access to secrets just by opening a PR

**Comment by hamelsmu** [comment]
@jlewi what do you think about my suggested changes above?  If you like them please commit and we can merge

**Comment by jlewi** [comment]
> Don't forget to refer to @master branch
I think I updated all the references

> While you are at it can you add a more verbose explanation to Issue_Triage/action/README.md so that the community can understand what business rules either add or remove an issue from the Kanban board?

Done

**Comment by jlewi** [comment]
> Ohhhhh ! !!!! I know why b/c you are opening this PR from a Fork. Yeah you cannot trigger a build from a Fork because your Fork doesn't have access to secrets and you cannot get access to secrets just by opening a PR

This seems like its working as intended. I would only expect the builds of the docker image to be triggered on postsubmit not presubmit. If we trigger on presubmit then people running the action might use unmerged code which seems problematic.

**Comment by jlewi** [comment]
@hamelsmu I think I addressed all your comments.

**Pull request merged**